code/modules/antagonists/wizard/equipment/soulstone.dm

```diff
@@ -246,7 +246,7 @@
 					T.mind?.remove_antag_datum(/datum/antagonist/cult)
 				if(theme == THEME_WIZARD)
 					icon_state = "mystic_soulstone2"
-				if(theme == THEME_CULT)
+				if(theme == THEME_FAKE_CULT)
```

Comment:
Like, stuff like this should accept theme == THEME_CULT, not just disregard it. the two are in theory equal themes, right?